doc/DataPlaneCodeGeneration/AzureSDKGeneration_Prerequistites.md

```diff
@@ -0,0 +1,16 @@
+# Prerequisites
```

Comment:
This file should include first time steps for a brand new package.

Reply:
I added the typespec project setup and sdk project folder setup in prequiesties.md